components/shopify_partner/sources/new_app_charges.js

```diff
@@ -0,0 +1,67 @@
+const shopify = require("../shopify_partner.app.js");
+const getAppTransactions = require("../queries/getAppTransactions");
+
+module.exports = {
+  key: "shopify_partner-new-app-charges",
+  name: "New App Charges",
+  type: "source",
+  version: "0.0.1",
+  description:
+    "Emit new events when new app charges made to your partner account.",
+  props: {
+    shopify,
```

Comment:
The `shopify` app prop, `db` and `timer` can be abstracted into a common file for all three sources

Reply:
Donezo